File: Android/java/doraemonkit/src/main/java/com/didichuxing/doraemonkit/widget/easyrefresh/exception/ERVHRuntimeException.java
package com.didichuxing.doraemonkit.widget.easyrefresh.exception;

/**
 * Created by guanaj on 16/9/21.
 */

public class ERVHRuntimeException extends RuntimeException {

    public ERVHRuntimeException() {
        super();
    }

    public ERVHRuntimeException(Throwable throwable) {
        super(throwable);
    }

    public ERVHRuntimeException(String detailMessage, Throwable throwable) {
        super(detailMessage, throwable);
    }

    public ERVHRuntimeException(String detailMessage) {
        super(detailMessage);
    }
}

File: Android/kotlin/doraemonkit/src/main/java/com/didichuxing/doraemonkit/widget/easyrefresh/exception/ERVHRuntimeException.kt
package com.didichuxing.doraemonkit.widget.easyrefresh.exception

/**
 * Created by guanaj on 16/9/21.
 */
class ERVHRuntimeException : RuntimeException {
    constructor() : super() {}
    constructor(throwable: Throwable?) : super(throwable) {}
    constructor(detailMessage: String?, throwable: Throwable?) : super(detailMessage, throwable) {}
    constructor(detailMessage: String?) : super(detailMessage) {}
}